src/promise-katas-2.js

```diff
@@ -25,17 +25,48 @@ fetch("joke", "question").then(<insert your callback function>)
 // 1 Create a function that uses the fetch function to make a request to the "food" URL and returns
 // the data - expected return value "Cheese" of type String
 
-const food = () => {};
+// THIS does work but uses the .then
+/* const food = async () => {
+  return await fetch("food")
+    .then(
+      value => { return value.data; }
+    );
+}; */
+
+//THIS works but does not use the .then
+const food = async () => {
+   const { data } = await fetch("food");
+   return data;
+ };
+
```

Comment:
Youre supposed to use the .then method, the instructions specify thats okay. Theres no need to write out the same function in 2 different ways, just one working version is fine.
In this case, Id keep the first one, but reword it a little as the standard placeholder value used for the result is `res` and not `value`. Also you dont need to use the `await` keyword here either, can remove it no problem:
```
const food = async () => {
  return fetch("food").then((res) => {
    return res.data;
  });
};
```